src/Tickets/Commerce/Gateways/Square/REST/Webhook_Endpoint.php

```diff
@@ -370,65 +383,29 @@ protected function process_order_event( array $event_data ) {
 			return;
 		}
 
-		$event_id = $event_data['id'] ?? '';
-
-		$option_name = 'tec_tc_webhook_' . $event_id;
-		$value       = microtime();
-		// This is a POS order, and we have no other "guard" in our system to prevent double processing.
-		// We will use a DB concat to prevent double processing.
-		$insert_statement = DB::prepare(
-			"INSERT INTO %i (option_name, option_value, autoload) VALUES (%s, %s, 'auto')",
-			DB::prefix( 'options' ),
-			$option_name,
-			$value
-		);
-
-		$select_statement = DB::prepare(
-			'SELECT option_value FROM %i WHERE option_name = %s',
-			DB::prefix( 'options' ),
-			$option_name
-		);
-
-		$delete_statement = DB::prepare(
-			'DELETE FROM %i WHERE option_name = %s AND option_value = %s',
-			DB::prefix( 'options' ),
-			$option_name,
-			$value
-		);
-
-		if ( 'order_created' === $type ) {
-			try {
-				DB::query( $insert_statement );
-
-				$values = DB::get_col( $select_statement );
-
-				if ( count( $values ) > 1 ) {
-					// This is a double event, so we skip.
-					DB::query( $delete_statement );
-					return;
-				}
-			} catch ( DatabaseQueryException $e ) {
-				// Insert query failed, so we skip.
-				return;
-			}
-		}
+		$event_id = $event_data['event_id'] ?? '';
```

Comment:
If `$event_id` is an empty string, you could just return here, right?

Reply:
done